app/models/alert_subscription_event.rb

```diff
@@ -9,6 +9,7 @@
 #  created_at            :datetime         not null
 #  id                    :bigint(8)        not null, primary key
 #  message               :text
+#  message_id            :uuid
```

Comment:
Consider splitting this out in to a separate object, like the find out more calculation object